modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1732,6 +1732,8 @@ public void testGetObjectEntries() throws Exception {
 				new ObjectEntry() {
 					{
 						properties = HashMapBuilder.<String, Object>put(
```

Comment:
You are appending the long value in an object entry belonging to `_objectDefinition1` and the assert below is performed over `_objectDefinition2`